src/var/isAttached.js

```diff
@@ -0,0 +1,11 @@
+define( [
+	"../core",
+	"../selector"
```

Comment:
Maybe add a comment that was previously in `curCSS`?
```
	"../selector" // Get jQuery.contains
```

Reply:
Done.